homeassistant/components/hassio/strings.json

```diff
@@ -19,11 +19,111 @@
   "issues": {
     "unhealthy": {
       "title": "Unhealthy system - {reason}",
-      "description": "System is currently unhealthy due to '{reason}'. Use the link to learn more about what is wrong and how to fix it."
+      "description": "System is currently unhealthy due to {reason}. Use the link to learn more and how to fix this."
+    },
+    "unhealthy_docker": {
+      "title": "Unhealthy system - Docker misconfigured",
+      "description": "System is currently unhealthy because Docker is configured incorrectly. Use the link to learn more and how to fix this."
+    },
+    "unhealthy_supervisor": {
+      "title": "Unhealthy system - Supervisor update failed",
+      "description": "System is currently unhealthy because an attempt to update Supervisor to the latest version has failed. Use the link to learn more and how to fix this."
+    },
+    "unhealthy_setup": {
+      "title": "Unhealthy system - Setup failed",
+      "description": "System is currently because setup failed to complete. There are a number of reasons this can occur, use the link to learn more and how to fix this."
```

Comment:
```suggestion
      "description": "System is currently unhealthy because setup failed to complete. There are a number of reasons this can occur, use the link to learn more and how to fix this."
```

Reply:
Also, what setup ? Is this initialization of the supervisor ?